css/main.css

```diff
@@ -0,0 +1,866 @@
+  }
+
+  .icon-tag {
+    background-position: -976px -976px;
+  }
+
+  .icon-briefcase {
+    background-position: -48px -48px;
+  }
+
+  .icon-facebook {
+    background-position: -272px -272px;
+  }
+
+  .icon-twitter {
+    background-position: -1024px -1024px;
+  }
+
+  .icon-youtube {
+    background-position: -1280px -1280px;
+  }
+
+  .icon-instagram {
+    background-position: -416px -416px;
+  }
+
+  .icon-pinterest {
+    background-position: -672px -672px;
+  }
+
+  .icon-rss {
+    background-position: -816px -816px;
+  }
+
+  .icon-mail {
+    background-position: -576px -576px;
+  }
+
+  #firstSection {
+    display: grid;
+    margin: auto;
+    grid-template-columns: 2fr 1fr;
+    grid-template-rows: 215px 215px;
+    grid-column-gap: 8px;
+    grid-row-gap: 8px;
+    color: white;
+  }
+
+  @media screen and (max-width: 768px) {
+
+    #firstSection {
+      display: grid;
+      margin: auto;
+      grid-template-columns: 1fr;
+      grid-template-rows:  150px 150px 150px;
+    }
+  }
+
+  .firstSectionTitle a {
+    color: white;
+    text-decoration: none;
+  }
+
+  #firstSection ul {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  #firstSection li {
+    margin-right: 5px;
+    display: flex;
+    font-size: 14px;
+  }
+
+  #coverDate {
+    position: relative;
+    bottom: 430px;
+    left: 82%;
+    width: 20%;
+    font-size: 15px;
+  }
+
+  .cover-gradient {
+    background: rgba(0, 0, 0, .25) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+    background-size: 100% 100%;
+    display: block;
+    height: 100%;
+    width: 100%;
+  }
+
+  .cover-gradient-small {
+    background: rgb(0, 0, 0, .05) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+    background-size: 100% 100%;
+    display: block;
+    height: 100%;
+    width: 100%;
+  }
+
+  .firstSectionLeft {
+    background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/07/Eventually-796x398.png") no-repeat;
+    background-size: 100% 100%;
+    grid-row: 1 / 3;
+  }
+
+@media screen and (max-width: 768px) {
+
+  .firstSectionLeft {
+    grid-row: 1 / 2;
+  }
+}
+
+  .firstSectionLeftLogo {
+    position: absolute;
+    background: transparent url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg") no-repeat;
+    background-size: contain;
+    left: 24px;
+    top: 110px;
+    height: 20%;
+    width: 25%;
+  }
+
+  .firstGridLeft {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 15px 50px 30px;
+    grid-row-gap: 15px;
+    position: relative;
+    bottom: 200px;
+    left: 10px;
+    width: 90%;
+    padding: 0px 10px;
+    align-items: center;
+  }
+
+  .firstGridLeft h5 {
+    margin-bottom: 60px;
+  }
+
+  .firstGridLeft ul {
+    margin-top: 50px;
+    color: gray;
+  }
+
+  .firstGridLeft h2 {
+    display: flex;
+    flex-direction: wrap;
```

Comment:
`SUGGESTION`

Consider changing to a correct value.

`flex-direction` does not have a `wrap` value. Possible values are `row` `row-reverse`, `column`, `column-reverse` and other global attributes. Learn more about `flex-direction` [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction)